identity-apps-core/apps/authentication-portal/src/main/webapp/retry.jsp

```diff
@@ -154,7 +154,17 @@
         <layout:component componentName="MainSection">
             <div class="ui orange attached segment mt-3">
                 <%
-                    if (StringUtils.equals(errorCode, IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE) &&
+                    if (IdentityCoreConstants.LOGIN_FAILED_GENERIC_ERROR_CODE.equals(errorCode)) {
+                %>
+                    <h3 class="ui header text-center slogan-message mt-3 mb-6">
+                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "unable.to.proceed")%>
```

Comment:
Shall we use the new `i18n` function to keep provision for custom text added via branding.
Do this for others as well.

```suggestion
                         <%= i18n(resourceBundle, customText, "unable.to.proceed") %>
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/7657/commits/85bd8bdeb287c4c700dde4c6c36b82c0f52f3f5a